cli/pkg/provider/state/customtype.go

```diff
@@ -178,7 +178,7 @@ func (s *CustomTypeState) FromResourceData(from resources.ResourceData) {
 	s.Description = MustString(from, "description")
 	s.Type = MustString(from, "type")
 	s.Config = MapStringInterface(from, "config", make(map[string]any))
-	s.Version = MustInt(from, "version")
+	s.Version = int(MustFloat64(from, "version"))
```

Comment:
Why is this casting necessary? Why would version be a float64?

Reply:
That's how go works, anything number added to interface{} will be typecasted by default into Float64.